prompt/async_git_prompt/prompt.nu

```diff
@@ -15,9 +15,10 @@ def prompt-git-branch [] {
 def prompt-create-left-prompt [] {
     let pwd = ($env.PWD | str replace $env.HOME '~')
     prompt-concat [
-        {text: $pwd, color: (ansi green_bold)}
-        {text: (prompt-git-branch), color: (ansi blue_bold)}
-        {text: (async-git-prompt-string), color: (ansi green_bold)}
+        [text color];
+        [$env.PWD (ansi green_bold)]
```

Comment:
`let pwd` is being set above. I'm not quite sure where we're not using it here in place of `$env.PWD`?

Reply:
Thanks! Done.